# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/android/Platform.cpp
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/android/Platform.cpp
--- a/accessible/android/Platform.cpp	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/android/Platform.cpp	Wed Nov 29 03:53:20 2023 +0000
@@ -193,24 +193,6 @@
 
 void a11y::PlatformSelectionEvent(Accessible*, Accessible*, uint32_t) {}
 
-void a11y::PlatformVirtualCursorChangeEvent(Accessible* aTarget,
-                                            Accessible* aOldPosition,
-                                            Accessible* aNewPosition,
-                                            int16_t aReason, bool aFromUser) {
-  if (!aNewPosition || !aFromUser) {
-    return;
-  }
-
-  RefPtr<SessionAccessibility> sessionAcc =
-      SessionAccessibility::GetInstanceFor(aTarget);
-
-  if (!sessionAcc) {
-    return;
-  }
-
-  sessionAcc->SendAccessibilityFocusedEvent(aNewPosition);
-}
-
 void a11y::PlatformScrollingEvent(Accessible* aTarget, uint32_t aEventType,
                                   uint32_t aScrollX, uint32_t aScrollY,
                                   uint32_t aMaxScrollX, uint32_t aMaxScrollY) {